Cart Flow/Base.lproj/LaunchScreen.storyboard

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
I know this is a WIP, but eventually you should set up your launch screen to display something reasonable during app startup so users aren't staring at a blank white screen. It could even just be your app icon centered in the screen. Beware of launch screen caching when you're testing this out...it may look like your changes aren't taking effect, even though they are. Here's how to clear the launch screen cache: https://rambo.codes/posts/2019-12-09-clearing-your-apps-launch-screen-cache-on-ios

Reply:
This is something I was planning to add, and was also considering putting together an onboarding process to help for time users get some initial setup in (adding stores/aisles etc.). Would not have considered the cache situation though, will be prepared!